pkg/client/helper.go

```diff
@@ -62,8 +63,20 @@ type Config struct {
 
 	// Server requires TLS client certificate authentication
 	CertFile string
-	KeyFile  string
-	CAFile   string
+	// Server requires TLS client certificate authentication
+	KeyFile string
+	// Trusted root certificates for server
+	CAFile string
```

Comment:
typically, CAFile (or the corresponding []byte sucked in from it) contains 1..N trusted root certs... serialized like
```
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
```